crates/chain/src/indexed_tx_graph.rs

```diff
@@ -27,12 +28,14 @@ impl<A, I: Default> Default for IndexedTxGraph<A, I> {
     }
 }
 
-impl<A: Anchor, I: Indexer> IndexedTxGraph<A, I> {
+impl<A, I> IndexedTxGraph<A, I> {
```

Comment:
I'm not sure how idiomatic this is, since we always end up using IndexedTxGraph with `A: Anchor, I: Indexer`, but I'm not opposed to this change if it's useful for us somehow

Reply:
Even if it's not useful directly, it may allow building logic around/above it not require as much generic constraints. I would leave it as is unless if there is downsides to this?